tests/conftest.py

```diff
@@ -143,6 +143,11 @@ def __init__(self, installs=[]):
     def get_installs(self):
         return self.installs
 
+    def get_install_to_run(self, tag):
+        company, _, tag = (tag.replace("/", "\\")).rpartition("\\")
```

Comment:
style: redundant brackets
```suggestion
        company, _, tag = tag.replace("/", "\\").rpartition("\\")
```